packages/cli/src/commands/deploy.ts

```diff
@@ -116,6 +116,14 @@ export default class Deploy extends AuthCommand {
       if (!preview) {
         await ux.wait(500)
         this.log(`Successfully deployed project "${project.name}" to account "${account.name}".`)
+
+        // Print the ping URL for heartbeat checks.
+        data.diff.forEach(async (check) => {
```

Comment:
Will `data.diff` only contains heartbeat checks?

Reply:
On the current implementation for the BE PR [here](https://github.com/checkly/checkly-backend/pull/4593), yes. But will most likely change the logic for this according to Chris's suggestion.